### ISSUE TITLE
Settings: Keyboard focus color is not clearly visible on Sort by precedence button

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#53136]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated: MAS2.4.7

**Repro Steps:**
1. Launch VS Code 
2. Open Keyboard Shortcuts from the Settings icon on the bottom left bar.
3. Apply Dark+(default dark) theme and other dark themes and check the focus visibility.

**Actual:**
Keyboard focus is not clearly visible on Sort by precedence button when default dark theme is applied. 

**Expected:**
Focus should be clearly visible and should be outlined by solid or dotted line.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to see the focus properly.

**MAS Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B3b98b04f-4fab-4d3e-81af-a0906a34849c%7D

Attachment for Reference: 
![issue2_keyboard](https://user-images.githubusercontent.com/39204909/41972836-b647f8a6-7a30-11e8-9230-1144189ddc7f.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Sounds like colors and contrast, also assigning to @misolori

**Comment by misolori** [comment]
@AccessibilityTestingTeam-TCS can you clarify if this also applies to our overall focus treatment? We use the same blue outline for most of our focus treatments:

![image](https://user-images.githubusercontent.com/35271042/42337620-32dcc656-803c-11e8-97b0-d6886d7ea03d.png)
![image](https://user-images.githubusercontent.com/35271042/42337631-379dd4fa-803c-11e8-8339-2c26b1d0e803.png)
![image](https://user-images.githubusercontent.com/35271042/42337673-5361e348-803c-11e8-9a69-c7d67983bd33.png)

**Comment by AccessibilityTestingTeam-TCS** [comment]
@misolori You can use any color for emphasizing the focus but it should be clearly visible to the user on all the color themes.

**Comment by sandy081** [comment]
@misolori Please let me know when there are any changes to be done from my side.

**Comment by misolori** [comment]
@sandy081 will do, I think I'll need to play around with the offset of the outline color (and possibly opacity).
